.ci/teamcity/oss/jest_integration.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source "$(dirname "${0}")/../util.sh"
+
+export JOB=kibana-oss-jest-integration
+
+checks-reporter-with-killswitch "OSS Jest Integration Tests" \
+  node scripts/jest_integration --verbose
```

Comment:
And this all the jest integration tests?

Reply:
Same, just OSS for now.